chcemvediet/templates/styleguide/components/sidebar.html

```diff
@@ -0,0 +1,19 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+
+{% comment %}
+ %
+ % Context:
+ %  -- None
+ %
+{% endcomment %}
+
+
+<h3>Sidebar</h3>
+<p>chv-nav</p>
+<div class="sg-specimen chv-scheme-white">
+  <ul class="nav nav-tabs nav-stacked chv-nav">
+    <li><a href="#">Inactive</a></li>
+    <li class="active"><a href="#">Active</a></li>
+    <li><a href="#">Dolorem</a></li>
```

Comment:
Chcel som, aby tam boli minimalne 3 prvky, ze je vidno `border-radius` a stredovy prvok.

Reply:
Kludne tam daj aj 5-6 riadkov. Nech vidno, ze to je menu.